addons/controls/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const ID = 'addon-controls' as const;
```

Comment:
To keep consistency with other addons, you should use `ADDON_ID` and `PARAM_KEY`